docs/high/file.rst

```diff
@@ -381,8 +381,8 @@ chunk cache*. They apply to all datasets unless specifically changed for each on
   ``rdcc_nbytes`` bytes. For maximum performance, this value should be set
   approximately 100 times that number of chunks. The default value is 521.
 
-Chunks and caching are described in greater detail in the `HDF5 documentation
-<https://portal.hdfgroup.org/display/HDF5/Chunking+in+HDF5>`_.
+Chunks and caching are described in greater detail in the `HDF5 documentation (TODO link)
```

Comment:
https://portal.hdfgroup.org/documentation/hdf5-docs/advanced_topics/chunking_in_hdf5.html ?

Reply:
portal -> support: https://support.hdfgroup.org/documentation/hdf5-docs/advanced_topics/chunking_in_hdf5.html. This link is also going to go away soon but let's use while it is still working. I will make this change.

Reply:
:face_exhaling: The portal links seem to be the ones that search engines find. If you want to change that, I believe that either HTTP 301 redirects or adding a `rel="canonical"` link should tell search engines which location is preferred.

Thanks!